R/utils.R

```diff
@@ -109,5 +136,8 @@ add_search_args <- function(...) {
   return(list(df))
 }
 
+encodeFixedPoints <- function(sourcePoint, targetPoints) {
```

Comment:
this should be a single `targetPoint`

Reply:
This encodes all target points at once, so no need to go over them one by one